src/main/java/cam72cam/immersiverailroading/tile/TileRailBase.java

```diff
@@ -533,6 +540,10 @@ protected void onContentsChanged() {
 				}
 			};
 			break;
+		case STEAM_LOADER:
+			this.augmentTank = new FluidTank(1000);
```

Comment:
Do you also need to filter this on Fluid_steam only?  Will it accept non fluid_steams after it's been emptied?

Reply:
yes, because it's the only way (that I found) I can accept Steam Fluids form other Mods.